examples/HOWTO.md

```diff
@@ -0,0 +1,45 @@
+# How to contribute examples
+
+This folder contain the pygfx example scripts.
+
+
+## Purpose
+
+Examples can serve a few purposes:
+
+* They demonstrate the capabilities of pygfx.
+* They explain how a particular visualization is performed (i.e. documentation).
+* They help make sure that pygfx does what it promises (i.e. testing).
+
+When creating / modifying an example, keep in mind how they're used.
+
+* People view them in the gallery docs (seeing a screenshot ot movie, but no interactivity).
+* People run the code, can interact (use mouse/keyboard) and modify the code.
+* The example is run in test, and when used as validation, the screenshot must be consistent.
+
+
+## Testing
+
+Examples have three possible modes with regards to testing. For this purpose the
+`# example_testing:` comment must be present in each file.
+
+* `# example_testing: no` - not tested.
+* `# example_testing: run` - tested by running the code (including rendering a frame).
+* `# example_testing: compare` - tested by running the code, rendering a frame, and comparing it to the reference (i.e. image comparisons).
```

Comment:
can you update the text in the README to refer to this explanation?

Reply:
It looks like we don't refer to this anywhere. (In wgpu we do refer to it in the readme.)

Reply:
I can add a note in the contributing.md